### PR TITLE
DevOpsInfrastructure SDK for .NET

### PR DESCRIPTION
**Pull request by oolaoluwa**
Followed steps outlined [here](https://eng.ms/docs/products/azure-developer-experience/develop/sdk-generate-mgmt-dotnet)

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@oolaoluwa:

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by oolaoluwa** [comment]
Just requested Azure SDK Partner access. Updated Codeowners

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -463,6 +463,9 @@
 # ServiceLabel: %DevOps
 # ServiceOwners:                                                   @narula0781 @ashishonce @romil07
 
+# PRLabel: %DevOpsInfrastructure
```
Comment (jsquire) [inline .github/CODEOWNERS:466]: As this is a management library, please move this to the management section around L942 and copy the format there to restrict the path pattern to the resource manager area.   Please update the label to be `DevOps - Infrastructure`, which I've created and added to the cross-repository common set.

Please be sure to maintain alphabetical ordering by tag.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -943,7 +943,7 @@
 # ServiceOwners:                                                   @davidemontanari @atastrophic @marcodalessandro @rohankhandelwal @riteshrao
 
 # PRLabel: %DevOps - Infrastructure
-/sdk/devops/Azure.ResourceManager.DevOpsInfrastructure/            @oolaoluwa
+/sdk/devopsinfrastructure/Azure.ResourceManager.*/                 @oolaoluwa
```
Comment (jsquire) [inline .github/CODEOWNERS:946]: Please add the `ServiceLabel` block for this label as well.

Comment (oolaoluwa) [reply .github/CODEOWNERS]: Like this?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -942,6 +942,9 @@
 # ServiceLabel: %Device Registry %Mgmt
 # ServiceOwners:                                                   @davidemontanari @atastrophic @marcodalessandro @rohankhandelwal @riteshrao
 
+# ServiceLabel: %DevOps - Infrastructure
+# ServiceOwners:                                                   @oolaoluwa
```
Comment (jsquire) [inline .github/CODEOWNERS:946]:
```suggestion
# PRLabel: %DevOps - Infrastructure
/sdk/devopsinfrastructure/Azure.ResourceManager.*/                 @oolaoluwa

# ServiceLabel: %DevOps - Infrastructure
# ServiceOwners:                                                   @oolaoluwa
```

Comment (jsquire) [reply .github/CODEOWNERS:946]: You want to have one entry for pull requests and another for issue triage.   This is the form that you need.

**Comment by jsquire** [comment]
Please work with @ArthurMa1978 and team for review of the SDK.

**Comment by oolaoluwa** [comment]
@ArthurMa1978 The Swagger PR has been merged

**Comment by oolaoluwa** [comment]
@ArthurMa1978 Any update?

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**